blog(embertimes116): octane super rentals (#280)
ember-learn/ember-blog: source/2019-09-20-the-ember-times-issue-116.md

---
title: The Ember Times - Issue No. 116
author: Chris Ng, the crowd
tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2019
alias : "blog/2019/09/20-the-ember-times-issue-116.html"
responsive: true
---

<SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
Check out the new Octane Super Rentals Tutorial 🚀,
<SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>

---

## [Octane Super Rentals Tutorial 🚀](https://octane-guides-preview.emberjs.com/release/tutorial/01-orientation/)

> [@chancancode](https://github.com/chancancode) and [@vaidehijoshi](https://github.com/vaidehijoshi) did some great work writing content and creating a tutorial code generator that runs commands, inserts the code into a tutorial, and even takes screenshots. You can now preview the [Octane guides](https://octane-guides-preview.emberjs.com/release/tutorial/01-orientation/) and check out the tutorial.

Reviewer: ```suggestion
[Godfrey Chan @chancancode](https://github.com/chancancode) and [Vaidehi Joshi @vaidehijoshi](https://github.com/vaidehijoshi) did some great work writing content and creating a tutorial code generator that runs commands, inserts the code into a tutorial, and even takes screenshots. You can now preview the [Octane guides](https://octane-guides-preview.emberjs.com/release/tutorial/01-orientation/) and check out the tutorial.
```

Author: I thought our conventions was to use the at github alias?

Reviewer: We've kinda been doing both lately e.g. https://blog.emberjs.com/2019/09/13/the-ember-times-issue-115.html. I guess someone did it and I thought it might be a bit more friendly

Reviewer: What do you think? Def don't have to keep going this way

Reviewer 2: Minor suggestion: maybe take out `and creating a tutorial code generator that runs commands, inserts the code into a tutorial, and even takes screenshots` in this first paragraph and combine it into the second one, so that the first paragraph is focused on content and the second one is focused on infrastructure.

Author: Updated the text based on @chancancode 's comment and ill accept this (@amyrlam 's change) for the non-alias name

Author: Oh I guess I can't accept it now so i'll push in a  new commit :)

> [@chancancode](https://github.com/chancancode) and [@vaidehijoshi](https://github.com/vaidehijoshi) did some great work writing content and creating a tutorial code generator that runs commands, inserts the code into a tutorial, and even takes screenshots. You can now preview the [Octane guides](https://octane-guides-preview.emberjs.com/release/tutorial/01-orientation/) and check out the tutorial.

Reviewer 2: There is a good chance that https://octane-guides-preview.emberjs.com/release/tutorial/00-part-1/ would have been [filled in](https://github.com/ember-learn/super-rentals-tutorial/pull/55) by the time we send this. If it's up, it would be great to change the link to the introduction chapter instead!

Reviewer: @chancancode ok, keep us posted. Are you saying that this link https://octane-guides-preview.emberjs.com/release/tutorial/00-part-1/ might be updated? I'm eyeing a post-4:30 PM pacific release

Reviewer 2: Yes! Working on it now!

Author: Cool will update the links in the blog

What’s amazing is that this tutorial is **automatically generated**. It runs the commands and edits the files the same way that the reader would. It even captures the output and screenshots. This way, if any steps stopped working due to a change in Ember itself, we would know right away from a CI failure.

> If you are looking to contribute check out the [super-rentals-tutorial repo on GitHub](https://github.com/ember-learn/super-rentals-tutorial)!

Reviewer 3: ```suggestion
If you are looking to contribute, check out the [super-rentals-tutorial repo on GitHub](https://github.com/ember-learn/super-rentals-tutorial)!
```

> If you are looking to contribute, check out the [super-rentals-tutorial repo on GitHub](https://github.com/ember-learn/super-rentals-tutorial)!

Reviewer 2: ```suggestion
If you are looking to contribute, or if you have any feedback on the content, check out the [super-rentals-tutorial repo on GitHub](https://github.com/ember-learn/super-rentals-tutorial)!
```

---

<SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
Check out the new Octane Super Rentals Tutorial 🚀,
the 1st EmberJS Meetup in Nizhny 🥳,
<SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>

---

## [Octane Super Rentals Tutorial 🚀](https://octane-guides-preview.emberjs.com/release/tutorial/00-part-1/)

> [Godfrey Chan @chancancode](https://github.com/chancancode) and [Vaidehi Joshi @vaidehijoshi](https://github.com/vaidehijoshi) did some great work writing new content for the revamped Super Rentals tutorial. You can now preview the [Octane guides](https://octane-guides-preview.emberjs.com/release/tutorial/00-part-1/) and check out the [tutorial](https://super-rentals-tutorial--ember-super-rentals.netlify.com/).

Reviewer 2: ```suggestion
[Godfrey Chan @chancancode](https://github.com/chancancode) and [Vaidehi Joshi @vaidehijoshi](https://github.com/vaidehijoshi) did some great work writing new tutorial content for Octane. You can now preview the [revamped Super Rentals tutorial](https://octane-guides-preview.emberjs.com/release/tutorial/00-part-1/) in the [Octane guides](https://octane-guides-preview.emberjs.com/).
```

I think _maybe_ it would be nice to link to the Super Rentals app too, but it seems hard to fit it in, and the tutorial text already links you to it.
